universal-application-tool-0.0.1/app/controllers/admin/AdminProgramBlockPredicatesController.java

```diff
@@ -106,4 +106,19 @@ public Result update(Request request, long programId, long blockDefinitionId) {
               "success", String.format("Saved visibility condition: %s", predicateDefinition));
     }
   }
+
+  @Secure(authorizers = Authorizers.Labels.UAT_ADMIN)
+  public Result destroy(long programId, long blockDefinitionId) {
```

Comment:
question (non-blocking): should we plan for removing only specific predicate nodes, instead of the entire predicate? Maybe not, I'm not sure how that would work since they don't have IDs. Maybe we can make an issue or todo instead

Reply:
Yep, I think let's leave this endpoint as being "totally clear it out" (especially because I think this will be useful even once we do have more complex predicates). And then we can make other methods that do smaller, more specific removals.